.CI/build_all

```diff
@@ -10,6 +10,7 @@ if [ -z "$GH_TOKEN" ]; then
     python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
+    conda config --add channels r
```

Comment:
Indeed. But my question here is if we want to do that or not? Should we package `r` itself and go from there? Or leverage our conda-forge with the "default" `r` channel.